ansible/inventory/group_vars/apps.yaml

```diff
@@ -74,25 +74,25 @@ messaging:
     contact_email: test@email.com
 
 compute:
-  key_pair_name: sandpit-key
+  key_pair_name: densikat-sandpit
```

Comment:
Please revert back to generic sandpit-key since it's just a sample default, so no specific user key as sample.